change/@ni-nimble-components-9da2496e-e544-4019-a666-d7fca3046771.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
With these changes, using up/down keyboard arrows no longer updates the scroll position of the listbox to keep the active element in view.

Reply:
Provided a new `focusAndScrollIntoView` method which we now call from the `toggleNewActiveOption` method.